### PR TITLE
feat: TE7-T1022: Implement profile screen UI

### PR DESCRIPTION
**Pull request by jigar-s-simform**
Implement profile screen UI

### CODE REVIEW

File: app/modules/profile/ProfileScreen.tsx [hunk 1]
```diff
@@ -1,9 +1,34 @@
...
-const ProfileScreen: FC = () => (
-  <Text>{Strings.bottomNavigationTitles.profile}</Text>
-);
+/**
+ * @description Profile screen component that will show the profile details of the user
+ * 
+ * @returns {JSX.Element}
+ */
+const ProfileScreen:FC = ():JSX.Element => {
+  const { userDetails } = useProfile();
```
Comment (sagar-simformsolutions) [inline app/modules/profile/ProfileScreen.tsx]: add return type

File: app/modules/profile/useProfile.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { authSelector, useAppSelector } from '../../redux';
+
+/**
+ * @description custom hook for implementing business logic
+ * 
+ * @returns {Object} containing user details
+ */
+const useProfile = () => {
```
Comment (sagar-simformsolutions) [inline app/modules/profile/useProfile.ts]: add return type

**Review by sagar-simformsolutions — CHANGES_REQUESTED** [review]

File: app/modules/profile/ProfileScreen.tsx (hunk 1 above)
Comment (jigar-s-simform) [reply app/modules/profile/ProfileScreen.tsx]: resolved

File: app/modules/profile/useProfile.ts (hunk 2 above)
Comment (jigar-s-simform) [reply app/modules/profile/useProfile.ts]: resolved

**Pull request merged**